converters/google/convert.go

```diff
@@ -180,19 +187,78 @@ func (c *Converter) AddResourceChanges(changes []tfplan.ResourceChange) error {
 		}
 
 		if rc.IsCreate() || rc.IsUpdate() || rc.IsDeleteCreate() {
-			if err := c.addCreateOrUpdate(rc); err != nil {
-				if errors.Cause(err) == ErrDuplicateAsset {
-					glog.Warningf("adding resource change: %v", err)
-				} else {
-					return errors.Wrapf(err, "adding resource change")
-				}
+			createOrUpdates = append(createOrUpdates, rc)
+		} else if rc.IsDelete() {
+			if err := c.addDelete(rc); err != nil {
+				return errors.Wrapf(err, "adding resource deletion")
```

Comment:
```suggestion
				return fmt.Errorf("adding resource deletion %w", err)
```

nit: `errors.Wrapf` is largely obviated by a core language feature as of `1.13`

Reply:
ah, great!

Reply:
Replaced this and a few other instances in this file in 41dd2648

Reply:
The one remaining usage is making the error a specific type, which seems like a different thing than what fmt.Errorf does?

Reply:
https://github.com/GoogleCloudPlatform/terraform-validator/pull/180/files#r577225621 should do the same thing using the core interface